### ISSUE TITLE
Braille Essentials 26.05.1 (emergency stable release)

### ISSUE THREAD

**Issue by josephsl** [nvaccess/addon-datastore#9027]
### Download URL

https://github.com/josephsl/brailleEssentials/releases/download/26.05/brailleEssentials-26.05.1.nvda-addon

### Source URL

https://github.com/josephsl/brailleEssentials

### Publisher

josephsl

### Channel

stable

### License Name

GPL v2

### License URL

https://www.gnu.org/licenses/gpl-2.0.html

**Comment by josephsl** [comment]
Changelog:

* NVDA will no longer fail to report spelling errors and other font attributes in braille when using NVDA 2026.1.
